### PR TITLE
Enable live ref tracking.

### PR DESCRIPTION
**Pull request by BraveMole**
This will kill server, but it's for the greater good.

Enable ref tracking when a harddel is detected. This will slow down server to a crawl, sadly. We are doing this because harddel are the bane of performances, and it will help us greatly to fix those

### CODE REVIEW

**Comment by Ieteihowmee** [comment]
Optimization good in the longer term.

File: code/_compile_options.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 //#define REFERENCE_TRACKING_DEBUG
 
 ///Run a lookup on things hard deleting by default.
-//#define GC_FAILURE_HARD_LOOKUP
+#define GC_FAILURE_HARD_LOOKUP
 #ifdef GC_FAILURE_HARD_LOOKUP
 #define FIND_REF_NO_CHECK_TICK
```
Comment (LemonInTheDark) [inline code/_compile_options.dm:70]: You're running this with no check tick? madman

Comment (BraveMole) [reply code/_compile_options.dm]: Oups, fixed

File: code/modules/admin/view_variables/reference_tracking.dm [hunk 2]
```diff
@@ -1,6 +1,14 @@
 #ifdef REFERENCE_TRACKING
 
+GLOBAL_LIST_INIT(reftracking_blacklisted_types, list(
+	/obj/effect/alien/weeds, //This is fixed in weeds refactor
+))
```
Comment (BraveMole) [inline code/modules/admin/view_variables/reference_tracking.dm:5]:
```suggestion
GLOBAL_LIST_EMPTY(reftracking_blacklisted_types)
```

**Comment by Al-1ce** [comment]
>break the game even more
>to fix the game
yeah no

**Pull request closed without merging**